src/pages/ServiceUnavailable/ServiceUnavailable.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import {Layout} from '../Layout/Layout';
+import styles from './ServiceUnavailable.module.scss';
+import image from '../../assets/images/broken_server.png'
+
+
+const ServiceUnavailable = () => (
+  <Layout>
+    <div className={styles['number-code']}>
+      <img src={image} alt={'broken_server.png'}/>
+      <h1>503</h1>
+      <h2>Service Unavailable</h2>
+      <body>Sorry, our servers are unavailable at the moment. We are actively working to get
+      them back up.</body>
```

Comment:
The HTML body tag should only be used once in an html document, and it's further up the chain than here.

I would wrap the text with a div instead if you aren't looking for margins. Don't forget to retarget your body selector. I would use a class instead to prevent changes of the html from affecting the css.

Also when writing a multiline html element, make sure you align the open and close tags.

```suggestion
      <div>
          Sorry, our servers are unavailable at the moment. We are actively working to get
          them back up.
      </div>
```

Reply:
Ohh okay I follow. I made the changes, lmk how it looks now.